lib/stat_tracker.rb

```diff
@@ -57,13 +57,39 @@ def percentage_visitor_wins
     (away_win?.to_f / @games.size).round(2)
   end
 
+  def percentage_ties
```

Comment:
This works perfectly but could potentially be shortened with something like this:
`def percentage_ties`
`tie_games = @game_teams.count { |game| game.game_result.include?("TIE") }`
`  (tie_games.to_f / @game_teams.length.to_f).round(2)`
`end`